.gcloudignore

```diff
@@ -40,7 +40,7 @@ Makefile
 # ui-codemirror, d3js, midi-js, ui-map, guppy, skulpt, math-expressions.
 # We exclude some of the scripts that are not directly imported in order to
 # reduce deployed file count.
-# TODO(sll): Find a more structured way of doing this.
+# TODO(#20275): Find a more structured way of doing this.
```

Comment:
This should be filed using the issue template format?

Reply:
Hi @seanlip, 

Thanks a lot for your response. 

I have updated the issue [#20275](https://github.com/oppia/oppia/issues/20275#issue-2283732244) using the proper template.  Kindly review it. Thank you.